fiona/ogrext3.pxd

```diff
@@ -322,6 +322,7 @@ cdef extern from "ogr_api.h":
     void *  OGROpenShared (char *path, int mode, void *x)
     int     OGRReleaseDataSource (void *datasource)
     OGRErr  OGR_L_SetIgnoredFields (void *layer, const char **papszFields)
+    OGRErr  OGR_L_SetAttributeFilter(void *layer, const char*)
```

Comment:
Thanks for doing this in all the .pxd files. In an upcoming PR we're going to delete ogrext1.pxd and ogrext2.pxd as we'll be dropping support for GDAL 1.x and 2.x.